lib/libc/gen/getpagesizes.c

```diff
@@ -27,11 +27,8 @@
  */
 
 #include <sys/param.h>
-#include <sys/mman.h>
-#include <sys/sysctl.h>
-
-#include <errno.h>
-#include <link.h>
+#include <sys/auxv.h>
+#include <sys/errno.h>
```

Comment:
You could commit this any time (I'm commenting early in a patch if I think I Can do that since I Don't see a way to talk about a commit otherwise.